recipes/nn.rb

```diff
@@ -19,6 +19,17 @@
   not_if { conda_helpers.is_upgrade || node["kagent"]["enabled"] == "false" }
 end
 
+exec = "#{node['ndb']['scripts_dir']}/mysql-client.sh"
+bash 'truncate_le_tables' do
+  user "root"
+  code <<-EOF
+      set -e
+      #{exec} -e \"TRUNCATE #{node['hops']['db']}.hdfs_le_descriptors\"
+      #{exec} -e \"TRUNCATE #{node['hops']['db']}.yarn_le_descriptors\"
```

Comment:
If you have multiple namenodes and running an upgrade, recipes might run at different time. What happens if the namenode is running, it's leader and someone truncates the table?

Reply:
@smkniazi Also, I suppose it's better if only one NameNode will run this block.

Reply:
Then this is not the right place to handle this. I will  handle it in the file system

Reply:
Closing PR